### DISCUSSION TITLE
Running example from documentation fails

### DISCUSSION THREAD

**Discussion by Odud** [hbldh/bleak#704] [Q&A] [answered]
* bleak version: **0.13.0**
* Python version: **3.10.0**
* Operating System: **Windows 10**
* BlueZ version (`bluetoothctl -v`) in case of Linux:

### Description

Install bleak and run a simple example from the doumentation

### What I Did

PS C:\Users\langb\OneDrive\GitHub\Bluetooth> python -m pip install bleak        
Collecting bleak
  Downloading bleak-0.13.0-py2.py3-none-any.whl (111 kB)
     |████████████████████████████████| 111 kB 3.3 MB/s
Collecting bleak-winrt>=1.0.1
  Downloading bleak_winrt-1.0.1-cp310-cp310-win_amd64.whl (340 kB)
     |████████████████████████████████| 340 kB 6.4 MB/s
Installing collected packages: bleak-winrt, bleak
Successfully installed bleak-0.13.0 bleak-winrt-1.0.1

Then used the code as shown in [https://bleak.readthedocs.io/en/latest/scanning.html](url)

```python
import asyncio
from bleak import BleakScanner

async def main():
    devices = await BleakScanner.discover()
    for d in devices:
        print(d)

asyncio.run(main())
```

Pylance shows an error _"discover" is not a known member of module,_ trying to run the code fails with _ImportError: cannot import name 'BleakScanner' from partially initialized module 'bleak' (most likely due to a circular import)_

**Comment by Odud** [comment]
The actual code in the exmples folder is slightly different, but still doesn't work

```python
import asyncio
from bleak import discover

async def main():
    devices = await discover()
    for d in devices:
        print(d)

if __name__ == "__main__":
    asyncio.run(main())
```

Pylance objects to the `devices = await discover()` line saying _Module is not callable_

**Comment by dlech** [comment, marked as answer]
It sounds like you have named your file `bleak.py` which causes the circular import error. Give your file a different name.

(same error as https://github.com/hbldh/bleak/discussions/516)

**Comment by Odud** [comment]
Ok thanks, stupid error I'll try again


Pete Barlow
